sos/plugins/ceph.py

```diff
@@ -37,14 +34,28 @@ class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
     )
 
     def setup(self):
+        if not self.get_option("all_logs"):
+            limit = self.get_option("log_size")
+        else:
+            limit = 0
+
+        if limit:
+            for f in (glob.glob("/var/log/ceph/*.log") +
+                      glob.glob("/var/log/radosgw/*.log") +
+                      glob.glob("/var/log/calamari/*.log")):
+                self.add_copy_spec_limit(f, limit)
```

Comment:
The `add_copy_spec()` and (now obsolete) `add_copy_spec_limit()` methods already understand globs - please don't expand them manually.